README.md

```diff
@@ -1,22 +1,28 @@
 # Machine-API Driven Remediation
 
-This operator conforms to the External Remediation of [NodeHealthCheck](https://github.com/medik8s/node-healthcheck-operator#readme) and is designed to work with [Node Health Check]((https://github.com/medik8s/node-healthcheck-operator#readme)) to reprovision unhealthy nodes using the [Machine API](https://github.com/openshift/machine-api-operator#readme). It functions by following the annotation on the Node to the associated Machine object, confirms that it has an owning controller (e.g. MachineSetController), and deletes it.  Once the Machine CR has been deleted, the owning controller creates a replacement. 
+This operator conforms to the External Remediation of [NodeHealthCheck](https://github.com/medik8s/node-healthcheck-operator#readme) and is designed to work with [Node Health Check](https://github.com/medik8s/node-healthcheck-operator#readme) to reprovision unhealthy nodes using the [Machine API](https://github.com/openshift/machine-api-operator#readme). It functions by following the annotation on the Node to the associated Machine object, confirms that it has an owning controller (e.g. MachineSetController), and deletes it.  Once the Machine CR has been deleted, the owning controller creates a replacement.
 
 ## Pre-requisites
+
 * Machine API based cluster that is able to programmatically destroy and create cluster nodes
 * Nodes are associated with Machines
 * Machines are declaratively managed
-* Node Health Check is installed and running 
+* Node Health Check is installed and running
 
 ## Installation
-- Deploy MDR (Machine-deletion-remediation) to a container in the cluster pod. Try `make deploy`, official images coming soon.
-- Load the yaml manifest of the MDR template (see below).
-- Modifying NodeHealthCheck CR to use MDR as it's remediator.
+
+* Deploy MDR (Machine-deletion-remediation) to a container in the cluster pod. Try `make deploy`, official images coming soon.
+
```

Comment:
Are this lines in between list items necessary?

Reply:
According to [MD032 - Lists should be surrounded by blank lines](https://github.com/DavidAnson/markdownlint/blob/v0.27.0/doc/md032.md) it is recommended.

Reply:
It seems to apply to groups of list items, not on each single item list